thriftpy2/http.py

```diff
@@ -99,6 +99,24 @@ def __init__(self, handler):
         self.handler = handler
 
 
+class THttpHeaderFactory(object):
+    """Default header factory return no custom headers
+    """
+    def __init__(self, headers=None):
+        """Initialize a header factory
+        @param headers(dict) a dictionary of static headers the factory generates
+        """
+        self.__headers = headers
+
+
+    def __enter__(self):
+        return self
+
+
+    def get_headers(self):
```

Comment:
I think it's better to ensure that `THttpHeaderFactory` will return a dict instead of None.

Reply:
Fixed in latest update